README.md

```diff
@@ -52,7 +52,7 @@ You will need the following:
 
 * JVM 8
 * Android SDK and the latest stable Android Studio or IntelliJ
-* Intellij Kotlin plugin with 1.3.71 support
+* Intellij Kotlin plugin with 1.3.71 support (should be included in the latest Android Studio)
```

Comment:
```suggestion
* Intellij Kotlin plugin with 1.3.72 support (should be included in the latest Android Studio or IDEA)
```

Reply:
Also, we should verify everything's working in AS 4.0 if we haven't yet. I'm usually in IDEA.